semver_tests/src/test_cases/struct_repr_transparent_removed.rs

```diff
@@ -114,6 +114,18 @@ pub struct WithGeneric {
     _marker: std::marker::PhantomData<&'static usize>,
 }
 
+#[cfg(not(feature = "struct_repr_transparent_removed"))]
+#[repr(transparent)]
+pub struct BothTransparent {
+    pub bar: usize,
+}
+
+#[cfg(feature = "struct_repr_transparent_removed")]
+#[repr(transparent, )]
```

Comment:
Oh, fascinating, I didn't realize this was legal! I'm assuming the semantics here are "trailing commas are ignored" and not something more devious?

Would it be worth adding such test cases to the other test files as well?

Reply:
This may be a problem, although I'm not sure yet, so I will create an issue about this.